desk/app/activity.hoon

```diff
@@ -590,7 +590,7 @@
   ::  we only care about posts/replies events that are notified, and we
   ::  don't want to include events from sources whose latest event is
   ::  after the start so we always get "new" sources when paging
-  ?.  ?&  notified.event
+  ?.  ?&  ?|(notified.event ?=(%contact -<.event))
```

Comment:
Seems like this circumvents the intended logic, but @arthyn approved this for now.

Reply:
yeah I think this is an ok compromise for now. what we really want are volume settings related to "where" an activity item should show up like unreads, feed, push notifications, etc. we had talked about it when first building activity, but it had already gone long so we decided to go with what we have.